src/components/GoToTop/GoToTop.test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
```

Comment:
Pretty annoying, but I had to `import * as React from 'react'` and not `import React from 'react'` because of the `jest.spyOn` on line 17.

Reply:
more detail on this. This is due to how we are importing `useState` on the actual component file.

Over there, we are importing it like `import React, {useState, etc etc} from 'react'`.

Due to this, the `spyOn` will not recognize the `useState`